### DISCUSSION TITLE
How implement Json Web tokens?

### DISCUSSION THREAD

**Discussion by aisakk** [AzureAD/azure-activedirectory-identitymodel-extensions-for-dotnet#2457] [Q&A]
Good afternoon, my question is the following, how can I create a token with System.IdentityModels.Tokens.jwt? in the documentation there is no clear example of how to implement them in a project, https://learn.microsoft.com/en-us/dotnet/api/system.identitymodel.tokens.jwt?view=msal-web-dotnet-latest I have the following documentation but it makes the tokens from the CLI https://learn.microsoft.com/en-us/aspnet/core/security/authentication/jwt-authn?view=aspnetcore-8.0&tabs=linux where I can get a clear example of the code where I specify each part?

**Comment by keegan-caruso** [comment]
If possible, you should prefer JsonWebToken over JwtSecurityToken.

For creating tokens use this API:

https://learn.microsoft.com/en-us/dotnet/api/microsoft.identitymodel.jsonwebtokens.jsonwebtokenhandler.createtoken?view=msal-web-dotnet-latest#microsoft-identitymodel-jsonwebtokens-jsonwebtokenhandler-createtoken(microsoft-identitymodel-tokens-securitytokendescriptor)
